### ISSUE TITLE
Picom Vsync Seems to get Ignored when Switching Between Groups and Returning to a Game

### ISSUE THREAD

**Issue by TheKarion** [yshui/picom#859]
### Platform

Endeavour OS, XFCE4 w/ Qtile as WM and Picom-git

### GPU, drivers, and screen setup

Nvidia 2080 w/ 5.15.57 (newest drivers)
name of display: :0.0
display: :0  screen: 0
direct rendering: Yes
Memory info (GL_NVX_gpu_memory_info):
    Dedicated video memory: 8192 MB
    Total available memory: 8192 MB
    Currently available dedicated video memory: 7311 MB
OpenGL vendor string: NVIDIA Corporation
OpenGL renderer string: NVIDIA GeForce RTX 2070/PCIe/SSE2
OpenGL core profile version string: 4.6.0 NVIDIA 515.57
OpenGL core profile shading language version string: 4.60 NVIDIA
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile

OpenGL version string: 4.6.0 NVIDIA 515.57
OpenGL shading language version string: 4.60 NVIDIA
OpenGL context flags: (none)
OpenGL profile mask: (none)

OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 515.57
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20

### Environment

XFCE + Qtile + Picom, but tested with just qtile and picom

### picom version
 
picom-git 1714_9.rc1.36

<details>
When playing a game in Fullscreen, switching between workspaces and returning to the game removes vsync, causing it to go above the refresh rate of my monitor and stutter a ton, the only way to fix it is to restart the game. Tested on TF2, Fallout New Vegas, and Elder Scrolls Oblivion
<summary>Diagnostics</summary>

</details>
[ 07/23/2022 14:26:37.719 parse_config_libconfig WARN ] The refresh-rate option has been deprecated. Please remove it from your configuration file. If you encounter any problems without this feature, please feel free to open a bug report
**Version:** vgit-896ac

### Extensions:

* Shape: Yes
* XRandR: Yes
* Present: Present

### Misc:

* Use Overlay: No
  (Another compositor is already running)
* Config file used: /home/domglusk/.config/picom.conf

### Drivers (inaccurate):

NVIDIA, modesetting

### Backend: glx

* Driver vendors:
 * GLX: NVIDIA Corporation
 * GL: NVIDIA Corporation
* GL renderer: NVIDIA GeForce RTX 2070/PCIe/SSE2

### Configuration:
<details>
    <summary>Configuration file</summary>

```
// Paste your configuration here
```
</details>

### Steps of reproduction

1. Launch any game in Fullscreen (not windowed borderless)
2. Switch to another group (meta shift 2) and switch back to the group with the game (meta shift 1) 

### Expected behavior
Game retains it's vsync and returns to just as I left it

### Current Behavior
Vsync get's broken, compositor doesn't seem to crash just ignores the game, not sure.

**Comment by yshui** [comment]
did you enable `unredir-if-possible`?

**Comment by TheKarion** [comment]
Yes I have, it seems to be specific to certain games, if launching from steam I can move it to an empty workspace where it sometimes will fix itself but not always, as with tf2 it sometimes works but most of the time I just have to restart

**Comment by yshui** [comment]
Some games set a bypassing compositor property, which picom will honor if you enabled `unredir-if-possible`. In that case picom will stop while the game is running.

**Comment by TheKarion** [comment]
I'll double check if it's disabled or not, I used the aur package picom-conf since it merges with xfce settings
